### DISCUSSION TITLE
É possivel manter o NavigationView quando mudamos para uma tela que não faz parte do NavigationBody ?

### DISCUSSION THREAD

**Discussion by Evandrogoes** [bdlukaa/fluent_ui#457] [Q&A]
Estou implementando um app onde na tela inicial eu tenho 4 botões que vão navegar para páginas que não fazem parte do NavigationView ! Como eu consigo manter a estrutura e apenas carregar o conteúdo ? é possível ?

**Comment by bdlukaa** [comment]
`Navigator.of(context).push(FluentPageRoute(builder: (context) => YourScreen()));`

https://pub.dev/documentation/fluent_ui/latest/fluent_ui/FluentPageRoute-class.html

**Reply by Evandrogoes** [reply]
Acredito que isso isoladamente não funcione. Eu fiz o teste e manteve o mesmo comportamento, não renderiza dentro do Panel.  Eu vi essa implementação no StackOverflow : https://stackoverflow.com/questions/71117371/navigation-inside-navigationbody-fluent-ui-flutter 

Realmente seria necessário algo desse nível de controle?

**Reply by bdlukaa** [reply]
Talvez você esteja procurando https://github.com/bdlukaa/fluent_ui/issues/265

**Reply by DreezDev** [reply]
I have the same problem, when I try to use the method you suggested or a simple "push" it returns the content correctly but not the navigation, so the user is stuck without being able to navigate to any other page due to the lack of a menu, is there a solution for this?

![image](https://user-images.githubusercontent.com/99101666/184668279-648d544e-0d9a-4299-acda-5ad4d6f3be22.png)

In addition to this, if I use:

Navigator.of(context).push(FluentPageRoute(builder: (context) => MyApp()));

It returns me correctly to the main.dart and if it shows me the navbar and the pages, but it does not work the same with the other components, example home.dart

![image](https://user-images.githubusercontent.com/99101666/184669342-2ae384b0-0c4c-41bd-be98-ce7361585b1d.png)
